src/Visual/Renderer.hs

```diff
@@ -44,3 +42,14 @@ positionPicture camera pic = Scale ratio ratio (Translate x y pic)
         offset = unwrap . invertVector . positionToVector . cameraPosition $ camera
         x = fst offset
         y = snd offset
+
+
+
+objectToPicture :: Object -> Picture
+objectToPicture obj = translate' obj $ scale' obj $ appearancePicture . objectAppearance $ obj
+  where scale' = \o -> uncurry Scale $ computeScale (appearanceBox . objectAppearance $ o)
```

Comment:
Why `scale’` is a lambda while getOffset is not?

Reply:
done